README.md

```diff
@@ -33,17 +33,22 @@ For detailed documentation, please refer to the [User Guide](docs/guide/user_gui
    mkdir -p cdr/cdm/jupyter/cdm_shared_workspace
    ```
 
-3. Create Docker network:
+3. Make scripts executable:
+   ```bash
+   chmod +x scripts/*.sh
```

Comment:
If you make the scripts executable locally and check them in they should be executable on clones. Works in my hands anyway

Reply:
👍

Reply:
Looks like the script is executable now so the instructions are no longer necessary

Reply:
👍